cypress/features/regression/accessibility/accessibilityExperimental.feature

```diff
@@ -69,13 +69,12 @@ Feature: Accessibility tests - Experimental folder
 
   @accessibility
   Scenario Outline: Experimental checkbox component <story> story
-    When I open "Experimental checkbox" component page "<story>" in no iframe
+    When I open "Experimental checkbox validations" component page "<story>" in no iframe
     Then "Experimental checkbox <story> story" component has no accessibility violations
     Examples:
       | story                              |
-      # Commented because of issue FE-3412
-       #| group checkbox boolean validation  |
-       #| group checkbox string validation   |
+      | group checkbox boolean validation  |
+      | group checkbox string validation   |
```

Comment:
Cypress tests run and pass but when I visit these stories on chrome it shows that there is an accessibility issue 
![image](https://user-images.githubusercontent.com/72866515/103277947-24834d80-49c2-11eb-904e-7445f7b44a84.png)

Reply:
it is because we've disabled `WGAC 2.0 a` and `WGAC 2.0 aa`

Reply:
ah had forgotten about this